src/services/item/plugins/app/chatBot/schemas.ts

```diff
@@ -1,33 +1,39 @@
+import { Type } from '@sinclair/typebox';
+import { StatusCodes } from 'http-status-codes';
+
 import { ChatbotRole, GPTVersion } from '@graasp/sdk';
 
-const chatBotDefinition = {
-  type: 'object',
-  required: ['role', 'content'],
-  properties: {
-    role: { type: 'string', enum: Object.values(ChatbotRole) },
-    content: { type: 'string' },
-  },
-};
+import { customType } from '../../../../../plugins/typebox';
+import { errorSchemaRef } from '../../../../../schemas/global';
 
-const create = {
-  body: {
-    type: 'array',
-    items: chatBotDefinition,
-    minItems: 1, // do not allow empty array
+const chatBotDefinition = Type.Object(
+  {
+    role: Type.Enum(ChatbotRole),
+    content: Type.String(),
   },
-  querystring: {
-    type: 'object',
-    properties: {
-      gptVersion: { type: 'string', enum: Object.values(GPTVersion) },
-    },
-    additionalProperties: false,
+  {
+    description: 'Chatbot',
   },
+);
+
+export const create = {
+  operationId: 'createAppChatbotPrompt',
+  tags: ['app', 'chatbot'],
+  summary: 'Create app chatbot prompt',
+  description: 'Create app chatbot prompt.',
+
+  body: Type.Array(chatBotDefinition, { minItems: 1 }),
+  params: customType.StrictObject({
+    itemId: customType.UUID(),
+  }),
+  querystring: customType.StrictObject({
+    gptVersion: Type.Optional(Type.Enum(GPTVersion)),
+  }),
```

Comment:
Add a description that lets the user know they can choose the model they want to use.